### ISSUE TITLE
Webform custom tab on service center

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#805]
@jlyon is there a way for the custom tab on the service center to pull an internal webform instead of an iframe like it is on SR?

SR Parks tried to do it but after saving he kept getting a bunch of additional ////'s

https://kettering-oh-parks.proudcity.com/wp-admin/admin.php?page=service-center-standalone
![screenshot 2017-01-25 13 36 19](https://cloud.githubusercontent.com/assets/6032426/22303978/9696631e-e303-11e6-873c-9a877cda2cd7.png)

**Comment by jlyon** [comment]
@kevindherman This will be fixed with the structure changes in #825.  I'll set up this form after the release.

**Comment by jlyon** [comment]
@kevindherman I set this up with the new system on K. Steps:

1. Create new Custom Tab post on https://kettering-oh.proudcity.com/wp-admin/edit.php?post_type=service_center_tab
2. Edit the Service Center homepage block, uncheck the (now empty) old Custom tab checkbox, check the new checkbox for the cusotm tab we just created:
![edit page kettering wordpress](https://cloud.githubusercontent.com/assets/550909/22542331/2bd0b2b8-e8e0-11e6-990a-29cd1924bb70.jpg)
